src/send_future.rs

```diff
@@ -32,7 +32,7 @@ pub enum ResolveToHandlerReturn {}
 /// Marker type for resolving the [`SendFuture`] to a [`Receiver`] once the message is queued into the actor's mailbox.
 ///
 /// The [`Receiver`] can be used to await the completion of the handler separately.
-pub enum ResoleToReceiver {}
+pub enum ResolveToReceiver {}
```

Comment:
The commit history would be cleaner if we would merge changes like this as separate PRs :)

Reply:
True

Reply:
I moved this to a commit on master since I think it's probably quite a noncontroversial change, so it doesn't need a whole branch and PR